src/CodeGenerators/SettingsGen/Wrappers/AttributeWrapper.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Omex.CodeGenerators.SettingsGen.Wrappers
+{
+	internal sealed class AttributeWrapper : IAttributeWrapper
```

Comment:
Why do you need `IAttributeWrapper` interface? And `IContextWrapper`?

Reply:
So the context and IAttribute wrapper were created in order to limit the usage of getting attributes and named symbols directly from GeneratorExecutionContext as its a readonly struct and there aren't many nice ways of testing it

Reply:
Still trying to understand it's usage, but if it's needed only for tests it should be internal

Reply:
Have made it internal

Reply:
Also the issue is mocking generatorsyntaxcontext, so in this case I am just putting the struct in a wrapper so that I can test getting the attributes without relying on the struct